azure-pipelines.yml

```
@@ -11,12 +11,18 @@ variables:
   value: Xamarin.Forms.sln
 - name: BuildVersion
   value: $[counter('$(Build.SourceBranchName)_counter', 1)]
-- name: MONO_VERSION
-  value: 5_18_1
 - name: XCODE_VERSION
   value: 10.2
 - name: NUGET_VERSION
-  value: 5.0.2
+  value: 5.2.0
+- name: DOTNET_VERSION
+  value: 3.0.100
```

```suggestion
  value: 2.1.801
```

CI currently doesn't support dotnet 3 until we have msbuild 16.3